draft-ietf-rtcweb-jsep.xml

```diff
@@ -3096,6 +3097,33 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
                 </section>
             </section>
+            <section title="Modifying an Offer or Answer"
+                anchor="sec.modifying-sdp">
+                <t>The SDP returned from createOffer or createAnswer MUST
+                NOT be changed before passing it to
+                setLocalDescription. If precise control over the SDP
+                is needed, the aformentioned createOffer/createAnswer options
+                or RTPSender APIs MUST be used.</t>
+
+                <t>Note that the application MAY modify the SDP to reduce the
+                capabilities in the offer it sends to the far side
+                (post-setLocalDescription) or the
+                offer that it installs from the far side (pre-setRemoteDescription),
+                as long as it retmains a
```

Comment:
I notice you just say offer here, but does this not apply to the answer as well?

I assume the argument here is that you can't modify the answer because the offerer uses that to determine what capabilities it must keep alive? If so, perhaps we should say that explicitly. Something like
"Applications MUST NOT modify the answer because it contains the answerer's representation to the offerer of what it intends to do."